src/Compilers/CSharp/Portable/Symbols/Source/LocalFunctionSymbol.cs

```diff
@@ -157,6 +157,15 @@ private void ComputeParameters()
 
             var diagnostics = DiagnosticBag.GetInstance();
             SyntaxToken arglistToken;
+
+            foreach (var param in _syntax.ParameterList.Parameters)
+            {
+                foreach (var attrList in param.AttributeLists)
+                {
+                    diagnostics.Add(ErrorCode.ERR_AttributesInLocalFuncDecl, attrList.Location);
```

Comment:
I don't know if we need to report on every single occurrence. Maybe stopping after the first error would be better?